### PR TITLE
Docs/improve readme

### PR DESCRIPTION
**Pull request by santinoncs**

### CODE REVIEW

**Review by rccrdpccl — COMMENTED** [review]
Although this is good in-depth documentation, I am missing a "Quick start" section where we could explain how to:
* Deploy app
* Write a node profile
* use nodeprofile
With very little text with instructions that can be copy&pasted

Another thing is that dev env is not necessarely kind. Although it's good to have an example of a specific dev env, I think we should clarify this.

**Comment by santinoncs** [comment]
Good point. I agree
Let me do that

File: README.md [hunk 1]
```diff
@@ -1,30 +1,104 @@
...
+
+## Deployment
 
-## DEVEL ENVIRONMENT
 
-### Requirements
+### Install k8s platform
 
-Install kind
+In this example we will use kind, but you can use whatever k8s platform you are comfortable with
```
Comment (rccrdpccl) [inline README.md]: I think we can move this step somewhere else out of the quick start. In the quick start we can assume we already have a k8s cluster. What do you think?

File: README.md [hunk 2]
```diff
@@ -1,30 +1,104 @@
...
+kind create cluster
 ```
 
+### Deploy using kubectl 
+
+
+Set the variables 
 
 ```bash
+VERSION ?= 0.1.1
```
Comment (rccrdpccl) [inline README.md]: No need to set this in Quick Start mode. Just get whatever there is. We can move this to a more in-depth section, what do you think?

File: README.md [hunk 3]
```diff
@@ -1,30 +1,104 @@
...
+Add the profile annotation to your pods, and the mutating webhook will take place
+
+Below you can see an extract of a deployment manifest
+
+```
+...
+  template:
+    metadata:
+      annotations:
+        nodepolicy.softonic.io/profile: "test"
```
Comment (rccrdpccl) [inline README.md]: maybe we should give a meaningful name to the example file

**Review by rccrdpccl — COMMENTED** [review]
Big improvement in the docs!!

File: README.md (hunk 1 above)
Comment (santinoncs) [reply README.md]: yes!

**Review by rccrdpccl — APPROVED** [review]

**Pull request merged**